easybuild/easyconfigs/l/LoRDEC/LoRDEC-0.9_GraphVector.patch

```diff
@@ -0,0 +1,73 @@
+diff -ru lordec-src_0.9.orig/lordec-correct.h lordec-src_0.9/lordec-correct.h
```

Comment:
Add description and author information for this patch.

Reply:
fixed in b9f6633